templates/base.html

```diff
@@ -51,13 +51,12 @@
 		</div><!-- /.container for main content -->
 
 		<footer class="footer">
-				<div class="container">
+				<div class="container">	
 					<a href="https://github.com/CodeCatz">
-					<img alt="CodeCatz" src="{% static 'CodeCatz.jpg' %}">
+						<img alt="CodeCatz" src="{% static 'CodeCatz.jpg' %}">
+						<p>GITHUB CodeCatz: Kittens learning to code like Catz</p>
 					</a>
-				</div>
+				</div>							
```

Comment:
would you mind removing the trailing whitespace? :wink: 
there are tabs after the </div>

also, this whole div (and it's content) might be indented one level less (to the left) but I will not hold that against you :angel: